src/MICore/JsonLaunchOptions.cs

```diff
@@ -185,17 +184,14 @@ public partial class LaunchOptions : BaseOptions
         /// <summary>
         /// The command to execute after the debugger is fully setup in order to cause the target process to run. Allowed values are "exec-run", "exec-continue", "None". The default value is "exec-run".
         /// </summary>
-        [JsonProperty("launchCompleteCommand", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        [JsonProperty("launchCompleteCommand", DefaultValueHandling = DefaultValueHandling.Ignore),
+        JsonConverter(typeof(LaunchCompleteCommandConverter))]
         public LaunchCompleteCommandValue? LaunchCompleteCommand { get; set; }
-
-        [JsonConverter(typeof(StringEnumConverter))]
+        
         public enum LaunchCompleteCommandValue
```

Comment:
>LaunchCompleteCommandValue [](start = 20, length = 26)

Now that you don't need this to make the serializer happy, I might suggest using the existing 'LaunchCompleteCommand' type.